client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -778,8 +1024,32 @@ class CancelPurchaseForm extends React.Component {
 
 	getStepButtons = () => {
 		const { flowType, translate, disableButtons, purchase, isImport } = this.props;
-		const { surveyStep } = this.state;
-		const disabled = disableButtons || this.state.isSubmitting;
+		const { isSubmitting, surveyStep } = this.state;
+		const disabled = disableButtons || isSubmitting;
+
+		if ( surveyStep === steps.FEEDBACK_STEP ) {
+			let actionText;
+			if ( flowType === CANCEL_FLOW_TYPE.REMOVE ) {
+				actionText = disabled ? translate( 'Removing…' ) : translate( 'Remove plan' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Remove Plan' )` *ES Score: 10*